src/models/GitTree.js

```diff
@@ -67,8 +68,8 @@ export class GitTree {
   _entries: Array<TreeEntry>
   */
   constructor (entries /*: any */) {
-    if (Buffer.isBuffer(entries)) {
-      this._entries = parseBuffer(entries)
+    if (isBuffer(entries)) {
+      this._entries = parseBuffer(Buffer(entries))
```

Comment:
Same here, `this._entries = parseBuffer(entries)` ought to work, and won't cause the linter to throw an error.